recipes-openxt/xenclient/xenclient-root-ro/init.root-ro

```diff
@@ -314,13 +314,6 @@ unlock_config()
     case "$sig" in
     LUKS)
         if [ -e /boot/system/tpm/enabled ]; then
-            [ "$measured" != 0 ] && {
-                maybe_break "txt-fail"
-                txt_failure
```

Comment:
txt_failure is now unused.  But maybe it would better to move the txt_failure error message after the unseal fails?  Display it conditionally depending on whether or not this is an EFI boot.

Reply:
Not possible unfortunately right now because the 32-bit kernel lacks the capacity to check if it's booted under UEFI. Once dom0 is upgraded to 64-bit that distinction can be made.

Reply:
Ah, okay.